package.json

```diff
@@ -391,7 +391,8 @@
           "description": "Specifies Lint tool name.",
           "enum": [
             "golint",
-            "gometalinter"
+            "gometalinter",
+            "megacheck"
```

Comment:
Hmm, I definitely tested that this was working before sending the PR in, seemed ok for me. Have a new computer setup since then though, so I'll get my env set back up and verify (hopefully later this evening EST).